app/presenters/user_analytics_presenter.rb

```diff
@@ -1,6 +1,5 @@
-class UserAnalyticsPresenter
+class UserAnalyticsPresenter < Struct.new(:current_facility)
```

Comment:
Can we put this back in an `initialize`? It's a bit plainer to read, and scalable too.

Reply:
These can scale with the number of arguments as well. I think a struct works here quite well since it's a decorator with only one object you are decorating over. 

This is a pretty standard pattern?

Reply:
I haven't used it much but it seems pretty neat. I don't feel strongly about it, let's go with `Struct` 👍